webpack/__mocks__/foremanReact/redux/API/index.js

```diff
@@ -1,4 +1,5 @@
-export default {
+export const post = data => ({ type: 'post-some-type', ...data });
+export const API = {
```

Comment:
maybe we should update all operations to the structure you added?

```suggestion
export default {
 post: data => ({ type: 'post-some-type', ...data });
}
```

though should we really test its functionality? I think it's more of a responsibility of Foreman

Reply:
I like to use it to see that the action sent the correct params, without this the action test will show `undefined`. I can add the rest of the operations (except delete) to this structure